serde/src/de/impls.rs

```diff
@@ -295,6 +297,33 @@ impl Deserialize for String {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+#[cfg(feature = "std")]
+impl Deserialize for Box<CStr> {
```

Comment:
Ah, that's unfortunate. This is basically a direct translation of `String::into_boxed_str`, since the equivalent `CString::into_boxed_cstr` method does not exist. I think it's more common (and useful) to have the `impl` for `CStr`... Also allows working with, say `Arc<CStr>` (or [arccstr](https://github.com/jonhoo/arccstr)). Should I then also remove the `Serialize` `impl`?